app/models/manageiq/providers/azure/regions.rb

```diff
@@ -139,8 +139,8 @@ module Regions
       # disabled regions.
       #
       def self.regions
-        additional_regions = Hash(Settings.ems.ems_azure.try!(:additional_regions)).stringify_keys
-        disabled_regions   = Array(Settings.ems.ems_azure.try!(:disabled_regions))
```

Comment:
Dan and I are not sure what the point of having the `!` here was. @agrare @Ladas - do you know? I want to make sure we are not eliminating some functionality through this PR.

Reply:
The semantics for `&.` and `try!` are the same, so this won't break anything, but I guess the question is why we're using `try!` instead of `try` in the first place.

Reply:
This looks like it was copied and pasted from my code in the amazon repo which I coped and pasted from Marcel haha
We aren't relying on the difference between `.try!` and `try`, `&.` should be fine here.